.github/workflows/pr-ci.yml

```diff
@@ -13,51 +49,55 @@ jobs:
-                                  libnuma-dev \
-                                  libudev-dev \
-                                  uuid-dev \
-                                  make \
-                                  ninja-build \
-                                  pandoc \
-                                  pkg-config \
-                                  python \
-                                  rpm \
-                                  sparse \
-                                  valgrind \
-                                  wget
+          sudo apt-get install -y ${{ env.APT_PACKAGES }}
       - uses: actions/checkout@v2
       - name: Build Check
         run: |
-          git clone --depth 1 -b v34.1 https://github.com/linux-rdma/rdma-core.git
+          set -x
+          git clone --depth 1 -b ${{ env.RDMA_CORE_VERSION }} https://github.com/linux-rdma/rdma-core.git
           pushd rdma-core; bash build.sh; popd
-          export RDMA_CORE_PATH="rdma-core/build"
-          export LD_LIBRARY_PATH="$RDMA_CORE_PATH/lib:$LD_LIBRARY_PATH"
+          export LD_LIBRARY_PATH="${{ env.RDMA_CORE_PATH }}/lib:$LD_LIBRARY_PATH"
           ./autogen.sh
-          ./configure --prefix=$PWD/install --enable-efa=${RDMA_CORE_PATH} \
-                                            --enable-mrail \
-                                            --enable-psm3=${RDMA_CORE_PATH} \
-                                            --enable-rxd \
-                                            --enable-rxm \
-                                            --enable-shm \
-                                            --enable-tcp \
-                                            --enable-udp \
-                                            --enable-usnic \
-                                            --enable-verbs=${RDMA_CORE_PATH} \
+          ./configure --prefix=$PWD/install ${{ env.OFI_PROVIDER_FLAGS }} CC=${{ matrix.cc }}
+          make -j 2; make install
+          $PWD/install/bin/fi_info -l
+  hmem:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Install dependencies (Linux)
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ${{ env.APT_PACKAGES }}
+      - name: Install CUDA
+        run: |
+          sudo apt-get install -y nvidia-cuda-toolkit
+      - name: Install ROCm
+        run: |
+          echo "Installing ROCm SDK"
+          # TODO: Install ROCm dependencies and add --with-rocm to build in next step
+      - name: Install Xe
+        run: |
+          echo "Installing Ze SDK"
+          # TODO: Install Ze dependencies and add --with-ze to build in next step
+      - uses: actions/checkout@v2
+      - name: HMEM Checks
+        run: |
+          set -x
+          # We could use 'upload-artifact' and persist the rdma-core build
+          # across jobs, but this is just as quick.
+          git clone --depth 1 -b ${{ env.RDMA_CORE_VERSION }} https://github.com/linux-rdma/rdma-core.git
```

Comment:
Since we but the version in an ENV, should we put the repo location?

Reply:
Made the version an env to make version bumps easier, but I did consider that with the repo URL given we won't change it.